kit/azure/buildingblocks/standard-vnet-configuration/main-backend.tf

```diff
@@ -14,35 +18,63 @@ data "azurerm_subscription" "sta_subscription" {
   subscription_id = local.sta_subscription_id
 }
 
+data "azurerm_storage_account" "tfstates" {
+  name                = local.sta_name
+  resource_group_name = local.sta_rg_name
+}
+
+# resource "azurerm_storage_container" "tfstates" {
+#   name = "tfstates-standard-vnet"
+#   storage_account_name = data.azurerm_storage_account.tfstates.name
+#   container_access_type = "blob"
+# }
```

Comment:
can you adding a comment why its commented out or removing it?

Reply:
We are creating the container using another provider here

Reply:
Agreed, please remove it (or add a comment why it's there but commented out, but if there is another provider, removing it seems better to me).